parser/SqlParser.ypp

```diff
@@ -676,7 +686,7 @@ sql_statement:
   | quit_statement {
     $$ = $1;
   }
-  | select_statement {
+  | set_operation_statement {
```

Comment:
What if we add `set_operation_statement`, but still keep `select_statement` so that we don't need to modify test files with `SELECT`-only queries?